cmd/e2e_test.go

```diff
@@ -41,5 +46,19 @@ func init() {
 func TestE2E(t *testing.T) {
 	klog.Info("Starting Red Hat App Studio e2e tests...")
 	gomega.RegisterFailHandler(ginkgo.Fail)
-	ginkgo.RunSpecs(t, "Red Hat App Studio E2E tests")
+	config, _ := ginkgo.GinkgoConfiguration()
```

Comment:
should we handle also the case if there any error in the config? 

```suggestion
	config, err := ginkgo.GinkgoConfiguration()
	if err != nil {
    klog.Errorf("Error loading Ginkgo configuration: %v", err)
    return
}
```

Reply:
Actually, GingkoConfiguration doesn't return an error. It returns a SuiteConfig and ReportConfig and the ReportConfig is what I'm ignoring. I've added a comment to make it clearer.